front_end/html/exercise.html

```diff
@@ -468,9 +474,28 @@ <h6 style="display: inline">Expected output:</h6>
                 </div>
                 {% if exercise_details["output_type"] == "txt" %}
                     <p><pre>{{ exercise_details["expected_text_output"] }}</pre></p>
-                {% else %}
+                {% elif exercise_details["expected_image_output"] != "" %}
                     <p><figure class="img"><img src="data:image/jpg;base64,{{ exercise_details['expected_image_output'] }}" width="100%"/></figure></p>
+                {% else %}
+                    <p><pre>[Expected solution does not produce an image.]</pre></p>
                 {% end %}
+
+                <div id="expected_tests">
+                {% if len(exercise_details["tests"]) > 0 %}
+                {% for i in range(len(exercise_details["tests"])) %}
```

Comment:
Add some vertical whitespace.

Should the `for` loop be indented within the `if` statement?